x-pack/plugins/infra/common/inventory_models/host/metrics/index.ts

```diff
@@ -59,23 +41,6 @@ export const hostSnapshotMetricTypes = Object.keys(exposedHostSnapshotMetrics) a
 >;
 
 export const metrics: InventoryMetrics = {
-  tsvb: {
```

Comment:
Does it make sense to attach the Lens formulas that are taking the place of the tsvb formulas to the host inventory model?  I understand we have the Lens formulas in another directory now, but I don't see why we'd move away from the "asset" type models.

Reply:
I think it makes sense to adapt the inventory model to support Lens formulas/chart. However, due to the fact the Lens and TSVB have different types,  we'll probably have to create a new property. 

Perhaps we could also move Lens formulas to the inventory models folder and this possible new property could return the content of [this folder](https://github.com/elastic/kibana/tree/main/x-pack/plugins/infra/public/common/visualizations/lens/dashboards/asset_details/host)

Since this looks like a big change it might be best to do it in a separate PR. wdyt?

Reply:
Was thinking the same.  Agree it should be in a separate issue and we can give it more thought.